cuda_core/cuda/core/experimental/_program.py

```diff
@@ -433,6 +503,27 @@ def _translate_program_options(self, options: ProgramOptions) -> LinkerOptions:
             ptxas_options=options.ptxas_options,
         )
 
+    def _translate_program_options_to_nvvm(self, options: ProgramOptions) -> list[str]:
```

Comment:
Looking good, could you also translate
- ftz
- prec_sqrt
- prec_div
- fma

and also support `-gen-lto`? This means in addition to `target_type="PTX"` we can also support `target_type="LTOIR"`